src/runtime_src/core/pcie/driver/linux/xocl/subdev/xgq.c

```diff
@@ -293,6 +294,31 @@ static bool xgq_submitted_cmds_empty(struct xocl_xgq *xgq)
 	return false;
 }
 
+static void xgq_vmr_log_dump(struct xocl_xgq_vmr *xgq)
+{
+	struct vmr_log log;
```

Comment:
Please initialize this structure to zeros.
```suggestion
	struct vmr_log log = {0};
```
Note: the '0' character is not needed, but having it helps defined what the intention is.

Reply:
fixed, thanks.